src/main/markdown/release-notes.md

```diff
@@ -62,6 +63,61 @@ The GWT Release Notes
 * * *
 
 <a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_9_0"></a> Release Notes for 2.9.0
+
+### Highlights
+- Able to compile projects with jsinterop-base 1.0.0, elemental2 1.0.0, and
+jsinterop-annotations 2.0.0. With the exception of @JsAsync and @JsEnum, this
+brings GWT2 to be compatible across these tools with J2CL.
+
+- Added support for Java language levels 9, 10, and 11.
+
+- Officially, support is dropped for running the GWT compiler or server-side
+tooling on Java 7. The GWT distribution is still compiled to run on Java 7
+for this release, but no guarantees are made about whether or not this will 
+work. Future versions will compile bytecode for Java 8+. The release was 
+tested and found to work cross platform when run with Java 8, 11, and 14.
+
+
+### Deprecations
+- Elemental has been officially deprecated - it is still included in this
+release, but may not appear in future releases. Instead, we encourage the use
+of the Elemental2 libraries, which are compatible with both GWT2 and J2CL.
+- Removed NoSuchMethodException emulation.
+
+### Bug fixes
+- Fixes Arrays.binarySearch semantics for float[] and double[]
+- Method references correctly implement their intersection cast
```

Comment:
You can remove this one. The description is not clear. IIRC this refers to the lambdas created by casting a method reference to an intersection cast now implement all types in the intersection.

Reply:
Removed.